OrganisationSample/src/main/java/services/FieldGeneration.java

```diff
@@ -0,0 +1,15 @@
+package services;
+
+import models.OrganisationEmployee;
+
+public class FieldGeneration {
+    private static FieldGeneration ourInstance = new FieldGeneration();
```

Comment:
were you trying to make it a singleton class. If yes then write a private constructor. You can also look into how you can make it thread safe.